tests/responses/tests.py

```diff
@@ -63,10 +63,22 @@ def test_change_status_code(self):
         self.assertEqual(resp.status_code, 503)
         self.assertEqual(resp.reason_phrase, "Service Unavailable")
 
+    def test_valid_status_code_string(self):
+        resp = HttpResponse(status='403')
+        self.assertEqual(resp.status_code, 403)
+
+    def test_invalid_status_code(self):
+        with self.assertRaisesMessage(TypeError, 'HTTP status code must be an integer'):
+            HttpResponse(status='bender')
+        with self.assertRaisesMessage(ValueError, 'HTTP status code must be an integer from 100 to 599'):
+            HttpResponse(status=22)
+        with self.assertRaisesMessage(ValueError, 'HTTP status code must be an integer from 100 to 599'):
+            HttpResponse(status=1729)
+
     def test_reason_phrase(self):
         reason = "I'm an anarchist coffee pot on crack."
-        resp = HttpResponse(status=814, reason=reason)
```

Comment:
think he was going for a reverse [418] 😃  (https://en.wikipedia.org/wiki/Hyper_Text_Coffee_Pot_Control_Protocol)